frontend/src/components/core/Button.tsx

```diff
@@ -53,43 +53,82 @@ export const TextButton = styled.button<ThemeType>`
   }
 `;
 
+export const RedTextButton = styled(TextButton)`
+  color: ${({ theme }) => theme.colors.red2};
+`;
+
+export const GrayTextButton = styled(TextButton)`
+  color: ${({ theme }) => theme.colors.regrayd2};
```

Comment:
Potential typo haha

```suggestion
  color: ${({ theme }) => theme.colors.gray};
```

Reply:
Nah it was just the second "regrayd" color 😁 Thanks for the fix haha